taui/src/actions/network.js

```diff
@@ -127,6 +128,9 @@ export const loadDataset = (
   dispatch(loadDataFromJSON('assets/neighborhoods.json', 'set neighborhoods'))
   dispatch(loadDataFromJSON('assets/neighborhood_bounds.json', 'set neighborhood bounds'))
 
+  // Start on first page of neighborhoods
+  dispatch(setPage(0))
```

Comment:
Is `dispatch` necessary here since it is called by `setPage`?

Reply:
I believe it's passing through `dispatch` to `setPage` to invoke (similar to `loadDataset` in `network.js`), so yes.